src/GameShelf.js

```diff
@@ -1,19 +1,27 @@
-import React from 'react';
+import React, { Component } from 'react';
 import GameCard from './GameCard';
 import './GameShelf.css';
 
-export default function Gameshelf(props) {
+export default class GameShelf extends Component {
+  constructor (props) {
+    super (props);
+    this.state = {}
```

Comment:
Giving gameshelf state, will hopefully allow us to implement library functionality later on. 
Or would it be better to execute that plan in the app.js?